tpm/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "tpm"
 version = "0.1.0"
 
 [dependencies]
-anyhow = "1.0.81"
+anyhow = "1.0.94"
```

Comment:
Would it help to declare common dependencies in the workspace root and inheriting them in workspace members? This way the versions have to be bumped in only one place: 

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table